README.md

```diff
@@ -13,15 +22,71 @@ notebooks in this repository:
 1. Install [Anaconda](https://docs.anaconda.com/anaconda/install/)
    or [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
 1. Navigate to your local copy of this repository.
-1. Run the script `env.sh` to create an Anaconda environment in the directory
-   `./env`:
+1. Run the script `env.sh` to create an Anaconda environment in the directory `./env`:
    ```console
    $ bash env.sh
    ```
    Note: This script takes a while to run.
 1. Activate the new environment and start JupyterLab:
    ```console
    $ conda activate ./env
-   $ jupyter lab
+   $ jupyter lab --debug
    ```
 
+#### Configuring a local Kubeflow Pipeline runtime
+
+[Elyra's Notebook pipeline visual editor](https://elyra.readthedocs.io/en/latest/getting_started/overview.html#notebook-pipelines-visual-editor)
+currently supports running these pipelines in a Kubeflow Pipeline runtime,
+and, if required, these are [the steps to install a local deployment of KFP](https://elyra.readthedocs.io/en/latest/developer_guide/deploying-kubeflow-locally-for-dev.html).
+
+After installing your Kubeflow Pipeline runtime, use the command below (with proper updates) to configure the new
+KFP runtime with Elyra.
+
+```bash
+elyra-metadata install runtimes --replace=true \
+       --schema_name=kfp \
+       --name=kfp-local \
+       --display_name="Kubeflow Pipeline (local)" \
+       --api_endpoint=http://cloning1.fyre.ibm.com:31380/pipeline \
```

Comment:
Hard-coded internal IBM IP addresses, port numbers, user names and passwords here need to be replaced with some instructions for users to obtain the relevant parameters from their cluster admins and enter those parameters in the appropriate place.